app/app.scss

```diff
@@ -1,5 +1,13 @@
 /* Used for app only. Do not deploy with library! */
 
+html {
+    font-size: 62.5%; /* 1rem = 10px */
```

Comment:
@ashkanx the `font-size` will be overwritten by `.smartcharts` class to `12px` therefore the values of `font-size` in each classes in `_chart-controls.scss` will be `12px` instead of `10px`. Also the values of left/right paddings will be `19.2px` instead of `16px`.